includes/misc-functions.php

```diff
@@ -380,15 +380,15 @@ function give_get_purchase_session() {
 }
 
 /**
- * Build item title for PayPal.
+ * Generate Item Title for Payment Gateway.
  *
  * @param array $payment_data Payment Data.
  *
- * @since 1.8
+ * @since 1.8.14
  *
  * @return string
  */
-function give_build_paypal_item_title( $payment_data ) {
+function give_payment_gateway_item_title( $payment_data ) {
```

Comment:
@mehul0810 can you write a unit test for this function so we're certain it works through the test conditions you listed?

Reply:
Sure @DevinWalker . I'll write unit tests and will update you.

Reply:
@mehul0810 any luck?

Reply:
@DevinWalker Yes... I have added unit test for simple donation forms. But, working on tests for Multi Level Donation for function `give_payment_gateway_item_title`. Updating you in some time.

Reply:
@DevinWalker I've added unit tests and make sure that it is working properly.